src/Module/Moderation/Blocklist/Server/Index.php

```diff
@@ -57,18 +57,19 @@ protected function post(array $request = [])
 
 		// Edit the entries from blocklist
 		$blocklist = [];
-		foreach ($request['domain'] as $id => $domain) {
+		foreach ((array)$request['domain'] as $id => $domain) {
 			// Trimming whitespaces as well as any lingering slashes
 			$domain = trim($domain);
 			$reason = trim($request['reason'][$id]);
-			if (empty($request['delete'][$id])) {
+			if (empty($request['delete'][$id]) && !empty($domain)) {
 				$blocklist[] = [
 					'domain' => $domain,
 					'reason' => $reason
 				];
 			}
 		}
 
+		$this->logger->debug('Blubb', ['blocklist' => $blocklist]);
```

Comment:
Please remove debug logging when you're done with a fix.

Reply:
Test code is removed